bindings/python/src/OpenSpaceToolkitAstrodynamicsPy/Trajectory/Orbit.cpp

```diff
@@ -52,10 +54,12 @@ inline void                     OpenSpaceToolkitAstrodynamicsPy_Trajectory_Orbit
             // .def("access_model", &Orbit::accessModel, return_value_policy<reference_existing_object>())
```

Comment:
@apaletta3 Let's remove the unused bindings to clean-up a bit?

Reply:
We could, but I'm not sure if this is something that Lucas wanted to implement originally and never got around to doing? what do you think? is it an important method that we should still try to impelement? @lucas-bremond @Derollez

Reply:
I think this is simply left over from the `Boost` to `Pybind11` transition basically (see uncommented function below)